File: src/test/java/org/openrewrite/kotlin/tree/ThrowTest.java
package org.openrewrite.kotlin.tree;

import org.junit.jupiter.api.Test;
import org.openrewrite.test.RewriteTest;

import static org.openrewrite.kotlin.tree.ParserAssertions.kotlin;

public class ThrowTest implements RewriteTest {

    @Test
    void returnValue() {
        rewriteRun(
          kotlin(
            """
            fun method ( ) {
                throw IllegalArgumentException ( "42" )
            }
            """
          )
        );
    }
}

File: rewrite-groovy/src/test/kotlin/org/openrewrite/groovy/tree/ThrowTest.kt
package org.openrewrite.groovy.tree

import org.junit.jupiter.api.Test

class ThrowTest : GroovyTreeTest {

    @Test
    fun throwException() = assertParsePrintAndProcess(
        """
            def test() {
                throw new UnsupportedOperationException()
            }
        """
    )
}
